pkg/kubelet/kubelet_volumes.go

```diff
@@ -111,7 +111,7 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(
 			continue
 		}
```

Comment:
Talk with @jsafrane offline, will address this issue in a separate PR. Opened up an issue to track this #41930